src/pages/wallet/CreateWallet.tsx

```diff
@@ -233,7 +252,32 @@ const CreateWallet = () => {
         style: 'decimal',
         minimumFractionDigits: getCurrencyPrecision(currency),
       }),
-      recurringTransactionRules: wallet?.recurringTransactionRules || undefined,
+      recurringTransactionRules:
```

Comment:
Is the the right moment to extract this to a deserialize method that would be independent and tested?